website/content/en/status/report-2023-04-2023-06/freebsd-foundation.adoc

```diff
@@ -0,0 +1,165 @@
+This included hosting the GSoC, link:https://wiki.freebsd.org/DevSummit/202305?action=AttachFile&do=view&target=FreeBSD_Foundation_Devsummit_Spring_2023_Day_2.pdf[FreeBSD Foundation] link:https://wiki.freebsd.org/DevSummit/202305?action=AttachFile&do=view&target=FreeBSD_Foundation_Devsummit_Spring_2023_Day_2_part1.pdf[Technical Review], and link:https://docs.google.com/presentation/d/e/2PACX-1vSnEW5Z0ttQOAeqEEY8KHkfiRGeFUm4i8XrYsfY8TNYD--yx1P6MUu2_u-mCcpe6PMMITjeDIgT31CC/pub[Workflow] working group sessions.
+Pierre Pronchery spoke about link:https://www.bsdcan.org/events/bsdcan_2023/schedule/speaker/89-pierre-pronchery/[driver harmony between the BSDs] and En-Wei Wu discussed link:https://www.bsdcan.org/events/bsdcan_2023/schedule/session/139-add-operating-modes-to-wtap4/[wtap work] completed under contract with the Foundation.
+
+==== Continuous Integration and Quality Assurance
+
+The Foundation provides a full-time staff member and funds projects to improve continuous integration, automated testing, and overall quality assurance efforts for the FreeBSD project.
+You can read more about CI work in a dedicated report entry.
+
+==== Advocacy
+
+Much of our effort is dedicated to the FreeBSD Project advocacy.
+This may involve highlighting interesting FreeBSD work, producing literature and video tutorials, attending events, or giving presentations.
+The goal of the literature we produce is to teach people FreeBSD basics and help make their path to adoption or contribution easier.
+Other than attending and presenting at events, we encourage and help community members run their own FreeBSD events, give presentations, or staff FreeBSD tables.
+
+The FreeBSD Foundation sponsors many conferences, events, and summits around the globe.
+These events can be BSD-related, open source, or technology events geared towards underrepresented groups.
+We support the FreeBSD-focused events to help provide a venue for sharing knowledge, working together on projects, and facilitating collaboration between developers and commercial users.
+This all helps provide a healthy ecosystem.
+We support the non-FreeBSD events to promote and raise awareness of FreeBSD, to increase the use of FreeBSD in different applications, and to recruit more contributors to the Project.
+We are grateful to be back to attending events mostly in person.
+In addition to attending and planning events, we are continually working on new training initiatives and updating our selection of link:https://freebsdfoundation.org/freebsd-project/resources/[how-to guides] to facilitate getting more folks to try out FreeBSD.
+
+Check out some of the advocacy and education work we did last quarter:
+
+* Helped to organize and attended the link:https://wiki.freebsd.org/DevSummit/202305[May 2023 Developer Summit] which took place May 17-18, 2023 in Ottawa, Ontario
+* Hosted a table and was the Tote Bag Sponsor of link:https://www.bsdcan.org/2023/[BSDCan], May 17-20, 2023 in Ottawa, Ontario.
```

Comment:
Was it May 17-18 or May 17-20?

Reply:
The developer summit was indeed May 17-18.  BSDCan ran until May 20.